core/ReleaseNotes.md

```diff
@@ -1,6 +1,83 @@
 Release Notes
 =============
 
+OpenRocket 22.xx Beta 1
+-----------------------
+(through PR1104)
+### Application
+* Update to Java xxx
+* Distributed as packaged installers
+
+### New Rocket Design Capabilities
+* **Pods!**
+* **Drop-off Boosters!**
+* **Rail buttons!**
+* **Attach freeform fins to nose cones and transitions!**
+* **Coefficient of Drag override!**
+* **Added Dave Cook's rocket component library!**
+
+### Staging Improvements
+* Update CG correctly when stages are toggled
+* Display stage names instead of numbers in Stage enable buttons
+* Reported length reflects length of selected stages only
+* More reliable deployment of booster stage recovery device
+* Fixed booster tumbling behavior
+* Eliminate exceptions during multi-stage simulation
+* Improved accuracy of nose cone simulation
+
+### Simulation Improvements
+* Assorted CP calculation fixes
+* Properly run all simulations when any design change is made
+* Use actual burn time for determination of burnout event
+* Improve mass calculation accuracy for motors
+* Correct linear interpolation of motor CG
+* Copy Simulation results to clipboard
+* Report wind speed correctly
+* Improved ground hit velocity calculations
+* Run simulations all the way to the end
+* No more warning if recovery device is deployed while motor is coasting
+
+### Rocket Appearance
+* Support rendering of transparent or translucent components
+* Separate inside and outside color for tube components
+* Separate left and right appearance for fin components
+
+### User Interface
+* Display rocket stability as percentage of length (preference)
+* Improved layout on many windows and dialogs
+* Now use "Export" to safe to RKT format
+* Improved File Dialog behavior
+* Multi-select/delete motor configurations
+* Improved UI appearance on Mac
+* Motor Selection
+  * Automatically open motor selection dialog when adding new motor configuation
+  * Added checkbox to hide motors which are out of production
+* Many improvements to freeform fin editor, including
+  * More accurate insertion of new points
+  * Highlight coordinates of selected point
+  * Export fin profile to CSV file
+  * Fixed scrolling and zooming
+* Improvements to Photo Studio
+  * Respond instantly to design changes
+  * Settings saved per rocket
+  * Settings layout improved
+  * Settings values can now be entered via keyboard
+
+### Misc
+* Updated motor list to latest data from Thrustcurve.org
+* Added additional fields to motors
+* Disable fin-thickness warnings on phantom tubes
+* Added warning message for phantom body tubes
+* Updated print dialog to allow simulation control
+* Improved Rocksim import
+* Translations:
+  * Added missing Spanish translation
+  * Added Dutch translation
+  * Refined simplified Chinese translation
```

Comment:
This is great! I also suspect there have been changes to the english text strings that have not propagated to the final translations. We'll need to iron out how we go about this for final release.

Reply:
I confess I know nothing about the details or status of the translations other than that there were PRs that claimed the above.  I should probably combine those lines into one (added or improved Spanish, Dutch, and Chinese translations).